betty/jinja2/__init__.py

```diff
@@ -214,6 +189,31 @@ def _init_i18n(self) -> None:
         )
         self.policies['ext.i18n.trimmed'] = True
 
+    @property
+    def context_class(self) -> type[Context]:  # type: ignore[override]
```

Comment:
Do we still need this change? And does the context class need to live within `Environment`, e.g. is it truly dynamic?